Index: apps/vue-storefront/src/themes/default/components/core/ProductQuantity.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- apps/vue-storefront/src/themes/default/components/core/ProductQuantity.vue	(date 1585214404125)
+++ apps/vue-storefront/src/themes/default/components/core/ProductQuantity.vue	(date 1585214404125)
@@ -3,8 +3,9 @@
     <base-input-number
       :name="name"
       :value="value"
-      :min="1"
+      :min="min"
       :max="max"
+      :step="step"
       :disabled="disabled"
       @input="$emit('input', $event)"
       @blur="$v.$touch()"
@@ -44,10 +45,18 @@
       type: Boolean,
       default: false
     },
+    minQuantity: {
+      type: Number,
+      default: 1
+    },
     maxQuantity: {
       type: Number,
       default: undefined
     },
+    stepQuantity: {
+      type: Number,
+      default: 1
+    },
     checkMaxQuantity: {
       type: Boolean,
       default: true
@@ -65,6 +74,9 @@
     isOnline (value) {
       return onlineHelper.isOnline
     },
+    min () {
+      return this.minQuantity
+    },
     max () {
       if (!this.isOnline || !this.isSimpleOrConfigurable) {
         return null
@@ -72,6 +84,9 @@
 
       return this.maxQuantity
     },
+    step () {
+      return this.stepQuantity
+    },
     disabled () {
       if (!this.isOnline) {
         return false
