Index: apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue	(date 1585238736971)
+++ apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue	(date 1585238736971)
@@ -6,6 +6,7 @@
       type="number"
       :min="min"
       :max="max"
+      :step="step"
       :disabled="disabled"
       class="m0 no-outline base-input-number__input brdr-cl-primary bg-cl-transparent h4"
       :focus="autofocus"
@@ -35,12 +36,16 @@
     },
     min: {
       type: Number,
-      default: 0
+      default: 1
     },
     max: {
       type: Number,
       default: undefined
     },
+    step: {
+      type: Number,
+      default: 1
+    },
     disabled: {
       type: Boolean,
       default: false
@@ -89,7 +94,7 @@
   &__input {
     border-style: solid;
     border-width: 0 0 1px 0;
-    width: 50px;
+    width: 60px;
     height: 1.4rem;
     line-height: 1.7rem;
     @media (min-width: 768px) {
