Index: apps/vue-storefront/src/modules/client.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- apps/vue-storefront/src/modules/client.ts	(date 1584968301673)
+++ apps/vue-storefront/src/modules/client.ts	(date 1584968301673)
@@ -25,7 +25,7 @@
   registerModule(CheckoutModule) // To Checkout
   registerModule(CartModule)
   registerModule(PaymentBackendMethodsModule)
-  registerModule(PaymentCashOnDeliveryModule)
+  // registerModule(PaymentCashOnDeliveryModule)
   registerModule(WishlistModule) // Trigger on wishlist icon click
   registerModule(NotificationModule)
   registerModule(UserModule) // Trigger on user icon click
