Index: apps/vue-storefront/src/themes/default/pages/Category.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- apps/vue-storefront/src/themes/default/pages/Category.vue	(date 1585212888273)
+++ apps/vue-storefront/src/themes/default/pages/Category.vue	(date 1585212888273)
@@ -144,20 +144,22 @@
     getCategoryProducts() {
       /* wrap getter to replace prices with group prices*/
       const user = this.$store.state.user.current;
-      const group_id = user.group_id;
       const result = this.$store.getters['category-next/getCategoryProducts'];
-      result.forEach((product, index, array) => {
-        if (Array.isArray(product.group_prices)) {
-          product.group_prices.forEach((one) => {
-            if (one.group_id == group_id) {
-              product.price = one.price;
-              product.price_incl_tax = one.price;
-              product.special_price = one.price;
-              product.special_price_incl_tax = one.price;
-            }
-          })
-        }
-      });
+      if (user && user.group_id) {
+        const group_id = user.group_id;
+        result.forEach((product, index, array) => {
+          if (Array.isArray(product.group_prices)) {
+            product.group_prices.forEach((one) => {
+              if (one.group_id == group_id) {
+                product.price = one.price;
+                product.price_incl_tax = one.price;
+                product.special_price = one.price;
+                product.special_price_incl_tax = one.price;
+              }
+            })
+          }
+        });
+      }
       return result;
     },
     isLazyHydrateEnabled () {
