Index: apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue	(date 1585211421383)
+++ apps/vue-storefront/src/themes/default/components/core/blocks/Form/BaseInputNumber.vue	(date 1585211421383)
@@ -6,6 +6,7 @@
       type="number"
       :min="min"
       :max="max"
+      :step="step"
       :disabled="disabled"
       class="m0 no-outline base-input-number__input brdr-cl-primary bg-cl-transparent h4"
       :focus="autofocus"
@@ -35,12 +36,16 @@
     },
     min: {
       type: Number,
-      default: 0
+      default: 1
     },
     max: {
       type: Number,
       default: undefined
     },
+    step: {
+      type: Number,
+      default: 1
+    },
     disabled: {
       type: Boolean,
       default: false
